src/coloring/bipartite.cpp

```
@@ -0,0 +1,84 @@
+            boost::color_traits <boost::default_color_type>::white() ?
+            results.push_back({{vid}, {0}}) : results.push_back({{vid}, {1}});
+    }
+    return results;
+}
+
+}  // namespace
+
+namespace pgrouting {
+namespace functions {
+
+std::vector<II_t_rt> pgr_bipartite(pgrouting::UndirectedGraph &graph ) {
```

Cppcheck (2.20.0)

[style] The function 'pgr_bipartite' is never used.

(unusedFunction)